packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -424,6 +424,11 @@ export interface ICompressionRuntimeOptions {
      * Compression is disabled if undefined.
      */
     readonly minimumSize?: number;
```

Comment:
Can you please clarify (in comment, and maybe in name of the variable) if size limit applies to single op or a batch of ops?

Also from new structure, I'd say both fields should be non-optional, but whole ICompressionRuntimeOptions can be optional. That eliminates some weird combos like - algorithm is specified, but no limit (probably fine, as it's same as INIFINITE), and limit specified, but no algorithm (this one seems like a combo that should not exist).

My proposal assumes that minimumSize = INIFINTE is Ok value to pass.

Reply:
I much prefer your proposal for both fields being non-optional. I hadn't thought of using INIFINTY as part of the default configuration, but that seems like it should work and removes awkward cases where one is defined and another is not. I've updated the docs comments to reflect this change and also renamed `minimumSize` to `minimumBatchSize` with a docs comment indicating what that value means.